features/unattached_commands.feature

```diff
@@ -529,6 +529,46 @@ Feature: Command behaviour when unattached
           | impish  |
           | jammy   |
 
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Unattached attach with json output
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I verify that running attach `as non-root` with json response exits `1`
+        Then I will see the following on stdout:
+            """
+            {"_schema_version": "0.1", "errors": [{"message": "This command must be run as root (try using sudo).", "service": null, "type": "system"}], "failed_services": [], "needs_reboot": false, "processed_services": [], "result": "failure", "warnings": []}
+            """
+        When I verify that running `ua attach invalid-token --format json` `with sudo` exits `1`
+        Then I will see the following on stdout:
```

Comment:
Although the JSON output goes to `stdout`, There is still `Invalid token. See https://ubuntu.com/advantage` being printed to `stderr`. Is that a problem? I think we should make sure only JSON is returned to wherever here

Reply:
Good catch, I will remove that print to stderr on that situation